addons/udes_delivery_control/models/picking_extra_fields.py

```diff
@@ -134,10 +95,11 @@ def _check_unloading_dates(self):
         msg = _("Unloading start time cannot be greater than end time")
         validate_dates(self.unloading_start_date, self.unloading_end_date, msg)
 
-    @api.constrains("backload_start_date", "backload_end_date")
-    def _check_backload_dates(self):
-        msg = _("Backloading start time cannot be greater than end time")
-        validate_dates(self.backload_start_date, self.backload_end_date, msg)
+    @api.constrains("loading_type", "backload_ids")
+    def _check_backload_ids(self):
+        if self.is_backload and not self.backload_ids:
+            msg = _("Please add backloads.")
+            raise ValidationError((msg) % self.backload_ids)
```

Comment:
Why do we need this ` % self.backload_ids` ?

Reply:
Removed